node/service_setup.go

```diff
@@ -48,7 +48,7 @@ func (node *Node) setupForShardValidator() {
 }
 
 func (node *Node) setupForBeaconLeader() {
-	nodeConfig, chanPeer := node.initBeaconNodeConfiguration()
+	nodeConfig, chanPeer := node.initNodeConfiguration(true, false)
```

Comment:
let's use the global configuration nodeconfig

Reply:
let's do it in a separate pr